README.md

```diff
@@ -41,6 +41,40 @@ Ask @acabunoc for ones marked `###########`. Our custom BadgeKit API code can be
 
 5. Run `npm start`, and open up `http://localhost:5000/` in your favourite web browser!
 
+### Using the Widget
+Scholars earn badges for their specific contributions to a scientific paper. A scholar who worked on investigation earns a prestigious investigation badge for that paper. 
```

Comment:
Can we tweak the language a bit to be more consistent?

Scholar -> researcher
scientific paper -> academic paper